Karen.java

```diff
@@ -0,0 +1,63 @@
+import com.sun.deploy.util.StringUtils;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Karen Xiao
+ * @see 5th Anniversary Class Report
+ * @since 11/9/18
+ */
+
+public class Karen
+{
+    private Integer age;
```

Comment:
"effective java" says you should use `final` for these :P

https://stackoverflow.com/questions/137868/using-the-final-modifier-whenever-applicable-in-java